### ISSUE TITLE
Catalina new issue:   Aerial would not exit back to OS when moving cursor, clicking trackpad, or hitting any key

### ISSUE THREAD

**Issue by jhewell56** [JohnCoates/Aerial#865]
- [ √] You have the latest version installed (There may be a beta version that fixes your issue), see here for the latest releases and bug fixes: https://github.com/JohnCoates/Aerial/releases
- [√ ] Your issue isn't already mentioned in our [issues]
- [ √] Your problem isn't mentioned in the [troubleshooting section]

- [ √] Mac model:  MacBookPro11,4
- [ √] macOS version: 10.15
- [ √] Monitor setup: Laptop

When using Catalina for first time, Aerial screensaver would not exit back to the OS as usual (e.g., when moving cursor, clicking trackpad, or hitting any key). I was able to exit screensaver only after repeated attempts.  Not sure if it is due to Photos background analysis.... Just figured I would note the issue.

**Comment by glouel** [comment]
@jhewell56 v1.6.2 comes with a fix in a few minutes (as soon as Apple notarizes it ;)). In the meantime uncheck "Auto install updates" in the Updates tab.

**Comment by jhewell56** [comment]
Thanks!

**Comment by glouel** [comment]
https://github.com/JohnCoates/Aerial/releases/download/v1.6.2/Aerial.saver.zip
